src/Workload/Microsoft.Maui.Sdk/Sdk/BundledVersions.in.targets

```diff
@@ -60,9 +60,6 @@
   </ItemGroup>
 
   <!-- These implicit <PackageReference/> pull dependencies from NuGet transitively -->
-  <ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' ">
-    <PackageReference Include="Microsoft.Maui.Extensions" Version="$(MauiVersion)" IsImplicitlyDefined="true" />
-  </ItemGroup>
```

Comment:
Put them here behind a condition:
```xml
<ItemGroup Condition=" '$(UseMaui)' == 'true' or '$(UseMauiCore)' == 'true' ">
```
Then your .apk size example wouldn't even get these packages at all.

It looks like this file is already setup to replace version numbers, so that part looks like it would work.

Reply:
Good idea, it seems like this should be functionally the same as before without an extra nupkg/msi.  These packages didn't declare IsImplicitlyDefined or a specific version before, but it seems like they should? I hard-coded the latest stable versions for now as it seems like the version wouldn't change often, it can be updated to do a package version string replace in the future if needed.

Reply:
Yes, I would put `IsImplicitlyDefined`. We could probably get rid of Microsoft.Maui.Dependencies, too, but maybe try that in a future PR.

We probably should use these properties instead of hardcoding those:

https://github.com/dotnet/maui/blob/64432c450080efa99084f040c933c9011b066989/eng/Versions.props#L7-L9

At some point in .NET 7, they might be updated by Maestro.

Reply:
Updated, I think this should be good to go. Similar changes for the Dependencies pack can happen in the future if we want.